From 9bde404768212581bc64bb4ebed965dadb280398 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:08:48 -0400
Subject: [usb] usb_wwan: make resume error messages uniform

Message-id: <1410887463-20674-457-git-send-email-dzickus@redhat.com>
Patchwork-id: 95593
O-Subject: [RHEL7 PATCH 456/591] USB: usb_wwan: make resume error messages uniform
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit b0f9d0030df8120713705c350fba8676f3556709
Author: Johan Hovold <jhovold@gmail.com>
Date:   Mon May 26 19:23:25 2014 +0200

    USB: usb_wwan: make resume error messages uniform

    Make resume error messages uniform.

    Signed-off-by: Johan Hovold <jhovold@gmail.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/serial/usb_wwan.c b/drivers/usb/serial/usb_wwan.c
index b671d59..3737006 100644
--- a/drivers/usb/serial/usb_wwan.c
+++ b/drivers/usb/serial/usb_wwan.c
@@ -672,8 +672,9 @@ int usb_wwan_resume(struct usb_serial *serial)
    urb = portdata->in_urbs[j];
    err = usb_submit_urb(urb, GFP_ATOMIC);
    if (err < 0) {
-    dev_err(&port->dev, "%s: Error %d for bulk URB %d\n",
-     __func__, err, i);
+    dev_err(&port->dev,
+     "%s: submit read urb %d failed: %d\n",
+     __func__, i, err);
     err_count++;
    }
   }
-- 
1.7.1